resolver/resolver.go

```diff
@@ -203,6 +203,15 @@ type State struct {
 // gRPC to add new methods to this interface.
 type ClientConn interface {
 	// UpdateState updates the state of the ClientConn appropriately.
+	//
+	// If an error is returned, the resolver should try to resolve the
+	// target again. Ideally the resolver uses a backoff timer to prevent
```

Comment:
Instead of `Ideally the resolver uses`, I think `The resolver should use` would be a bit more appropriate.

Reply:
changed